source/features/faster-pr-diff-options.tsx

```diff
@@ -3,14 +3,14 @@ import select from 'select-dom';
 import * as icons from '../libs/icons';
 import features from '../libs/features';
 
-function createDiffStyleToggle() {
+function createDiffStyleToggle(): JSX.Element {
```

Comment:
I'm confused, why does `HTMLElement` work in `getDefaultBranchLink` but not here? They both use JSX

Reply:
As mentioned above, `JSX.Element` is simply a type alias for `HTMLElement | SVGElement` which is why it works above.